src/OpenSpaceToolkit/Astrodynamics/Conjunction/Messages/CCSDS/CDM.cpp

```diff
@@ -0,0 +1,747 @@
+
+    // Extract Conjunction Objects Data
+
+    // Extract Object 1 Data
+
+    // Construct Object 1 Covariance Matrix, TBI: Use symmetric construction
+
+    MatrixXd object1CovarianceMatrix = MatrixXd::Zero(9, 9) ;
+    object1CovarianceMatrix(0, 0) = std::stod(aDictionary["SAT1_CR_R"].accessString()) ;
+    object1CovarianceMatrix(1, 0) = std::stod(aDictionary["SAT1_CT_R"].accessString()) ;
+    object1CovarianceMatrix(1, 1) = std::stod(aDictionary["SAT1_CT_T"].accessString()) ;
+    object1CovarianceMatrix(2, 0) = std::stod(aDictionary["SAT1_CN_R"].accessString()) ;
+    object1CovarianceMatrix(2, 1) = std::stod(aDictionary["SAT1_CN_T"].accessString()) ;
+    object1CovarianceMatrix(2, 2) = std::stod(aDictionary["SAT1_CN_N"].accessString()) ;
+    object1CovarianceMatrix(3, 0) = std::stod(aDictionary["SAT1_CRDOT_R"].accessString()) ;
+    object1CovarianceMatrix(3, 1) = std::stod(aDictionary["SAT1_CRDOT_T"].accessString()) ;
+    object1CovarianceMatrix(3, 2) = std::stod(aDictionary["SAT1_CRDOT_N"].accessString()) ;
+    object1CovarianceMatrix(3, 3) = std::stod(aDictionary["SAT1_CRDOT_RDOT"].accessString()) ;
+    object1CovarianceMatrix(4, 0) = std::stod(aDictionary["SAT1_CTDOT_R"].accessString()) ;
+    object1CovarianceMatrix(4, 1) = std::stod(aDictionary["SAT1_CTDOT_T"].accessString()) ;
+    object1CovarianceMatrix(4, 2) = std::stod(aDictionary["SAT1_CTDOT_N"].accessString()) ;
+    object1CovarianceMatrix(4, 3) = std::stod(aDictionary["SAT1_CTDOT_RDOT"].accessString()) ;
+    object1CovarianceMatrix(4, 4) = std::stod(aDictionary["SAT1_CTDOT_TDOT"].accessString()) ;
+    object1CovarianceMatrix(5, 0) = std::stod(aDictionary["SAT1_CNDOT_R"].accessString()) ;
+    object1CovarianceMatrix(5, 1) = std::stod(aDictionary["SAT1_CNDOT_T"].accessString()) ;
+    object1CovarianceMatrix(5, 2) = std::stod(aDictionary["SAT1_CNDOT_N"].accessString()) ;
+    object1CovarianceMatrix(5, 3) = std::stod(aDictionary["SAT1_CNDOT_RDOT"].accessString()) ;
+    object1CovarianceMatrix(5, 4) = std::stod(aDictionary["SAT1_CNDOT_TDOT"].accessString()) ;
+    object1CovarianceMatrix(5, 5) = std::stod(aDictionary["SAT1_CNDOT_NDOT"].accessString()) ;
+
+    for (int i = 0; i < 5; i++)
+    {
+        for (int j = 1; j < 6; j++)
+        {
+            object1CovarianceMatrix(i, j) = object1CovarianceMatrix(j, i) ;
+        }
+    }
+
+    CDM::Data sat1ObjectData = CDM::Data
+    {
+        Instant::Undefined(),
+        Instant::Undefined(),
+        Duration::Undefined(),
+        Duration::Undefined(),
+        Integer::Undefined(),
+        Integer::Undefined(),
+        Integer::Undefined(),
+        Integer::Undefined(),
+        Real::Undefined(),
+        Real::Undefined(),
+        Real::Undefined(),
+        Real::Undefined(),
+        Real::Undefined(),
+        Mass::Undefined(),
+        Real::Undefined(),
+        Real::Undefined(),
+        Real::Undefined(),
+        Real::Undefined(),
+        State(
```

Comment:
```suggestion
        State
        (
```
 if we want to keep Allman consistency + apply to rest